portal/types/stakingDashboard.ts

```diff
@@ -49,6 +43,18 @@ export type StakingPosition = {
   transferable: boolean
 }
 
+export type StakingDashboardOperation = Partial<{
+  approvalTxHash: Hash
+  input: string
+  inputDays: string
+  lockupDays: number
+  transactionHash: Hash
+  stakingPosition: Partial<
+    Pick<StakingPosition, 'amount' | 'tokenId' | 'lockTime' | 'timestamp'>
+  >
+  status?: StakingDashboardStatus
```

Comment:
A minor comment - This can be removed, as there's already a `Partial`
```suggestion
  status: StakingDashboardStatus
```